src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+Microsoft.CodeAnalysis.VisualBasic.Syntax.SyntaxNodeRemover.New() -> Void
```

Comment:
@jasonmalinowski @shyamnamboodiripad Addressed this with https://github.com/dotnet/roslyn/pull/10312/commits/017fcb958bbb32af32e5f82664c92a9a56587e4f